apps/settings/js/panels/addons/addons_list.js

```diff
@@ -9,16 +9,22 @@ define(function(require) {
   var SettingsService = require('modules/settings_service');
   var ListView = require('modules/mvvm/list_view');
   var AddonItemTemplateFactory = require('panels/addons/addons_template');
+  var ObservableArray = require('modules/mvvm/observable_array');
 
   function AddonsList(root, manager) {
     this._enabled = false;
     this._listRoot = root;
     this._addonManager = manager;
+    this._addons = ObservableArray(this._addonManager.addons.array);
```

Comment:
Let's initialize it with an empty array and call to `_update` for the real initialization.